src/server/api/vice.js

```diff
@@ -148,5 +148,35 @@ export default function viceRouter() {
         })
     );
 
+    logger.info("adding the GET /vice/listing handler");
```

Comment:
Small thing, but the log doesn't match the actual path being called

Reply:
D'oh, thanks.